Screen blur fix when swap pending

vinuman opened the pull request:
closes #168

ajayjoseaji commented:
@vinuman https://amal-jose.neetorecord.com/watch/a9297a924c38a6fb2000

ajayjoseaji commented on libs/web/src/components/common/Swap/Swap.tsx:
```diff
@@ -641,7 +647,10 @@ const Swap = ({isWidget}: {isWidget?: boolean}) => {
           <div className={styles.splitter}>
-            <IconButton onClick={swapAssets} className={styles.convertButton}>
+            <IconButton
+              onClick={swapPending ? () => {} : swapAssets}
```
@vinuman What about adding a `isDisabled` prop instead?

vinuman commented:
@ajayjoseaji 

The navbar was not blurred earlier as well. So the user was able to navigate away when swap is pending. This fix might need another direction of work. I have updated all other changes.

1. Added props to disable the Max text button in the Currency Box component
2. Added prop to disable Icon button component when swap is pending
3. Added logic so that the covert svg does not spin on hover when swap is pending

ajayjoseaji commented on libs/web/src/components/common/SlippageSetting/SlippageSetting.tsx:
```diff
@@ -6,15 +6,23 @@ import styles from "./SlippageSetting.module.css";
+  isDisabled,
+}: Props) => {
+  const handleClick = () => {
+    if (!isDisabled) openSettingsModal();
```
@vinuman I think we can now pass the isDisabled prop directly to the IconButton component. How about using that instead?

ajayjoseaji commented on libs/web/src/components/common/SlippageSetting/SlippageSetting.tsx:
```diff
@@ -6,15 +6,23 @@ import styles from "./SlippageSetting.module.css";
         {slippage / 100}% Slippage
       </p>
-      <IconButton onClick={openSettingsModal} className={styles.settingsButton}>
+      <IconButton onClick={handleClick} className={styles.settingsButton}>
```
passing `isDisabled` here ?

ajayjoseaji reviewed (approved):
Looks good
